### PR TITLE
feature/SOF-6597-2

### PR DESCRIPTION
**Pull request by VsevolodX**
Handling of JSON and POSCAR files. Although when parsing POSCAR lattice type is set to default ("TRI") now, no automatic detection.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -4,3 +4,4 @@ node_modules/
 .idea/**
 .husky
 .nyc_output
+.DS_Store
```
Comment (unsigned6) [inline .gitignore:7]: https://dev.to/lpheller/quickly-ignore-dsstore-files-globally-ecd

**Review by unsigned6 — COMMENTED** [review]
Looks good

File: src/parsers/native_formats.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+            return Poscar.fromPoscar(text);
+        case NATIVE_FORMAT.UNKNOWN:
+            throw new Error(`Unknown format`);
+        // TODO:  add more formats
+        default:
+            throw new Error(`Unsupported format: ${format}`);
+    }
+}
+
+export default {
```
Comment (k0stik) [inline src/parsers/native_formats.js:49]: You already use naming export for `export const NATIVE_FORMAT`
Let's have a naming export for `detectFormat` and `convertFromNative` then:
`export function detectFormat() ...`
`export function detectFormat() ...`

Comment (timurbazhirov) [reply src/parsers/native_formats.js:49]: To be using default export

**Review by timurbazhirov — APPROVED** [review]

**Pull request merged**